src/app/ubs/ubs-admin/components/shared/components/time-picker/time-picker.component.ts

```diff
@@ -75,6 +77,16 @@ export class TimePickerComponent implements OnInit {
     return arr;
   }
 
+  compareToTime(): string[] {
+    const arr = [];
+    this.toSelect.forEach((val, index) => {
+      if (this.currentHour < this.toSelect[index]) {
+        arr.push(this.toSelect[index + 1]);
+      }
+    });
+    return arr;
+  }
```

Comment:
remove forEach, use method filter here:

return this..toSelect.filter(item => item > this.currentHour)